plenum/server/replica.py

```diff
@@ -861,7 +912,7 @@ def processReqDuringBatch(
 
     @measure_replica_time(MetricsName.CREATE_3PC_BATCH_TIME,
                           MetricsName.BACKUP_CREATE_3PC_BATCH_TIME)
-    def create3PCBatch(self, ledger_id):
+    def create_3pc_batch(self, ledger_id, is_empty=False):
```

Comment:
Do we really need this additional parameter? If I got it correctly setting it to `True` will ask to create an empty batch even if there are some transactions in queue. I understand that this low probability situation might happen if freshness check happens just after receiving some write requests (less than max batch size), but before max batch wait time expired, however I don't see anything bad in actually ordering some transactions along.

Reply:
Agree, we don't need this, so it can be safely removed. Thanks for finding,